lib/headers.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function getHeaders (defaults) {
+  var headers = chrome(random(require('./browsers').chrome));
+  return Object.assign({}, defaults, headers);
+};
+
+function random (arr) {
+  if (Array.isArray(arr)) {
+    return arr[Math.floor(Math.random() * arr.length)];
+  }
+
+  return arr;
+}
+
+function chrome (options) {
+  options.headers['Connection'] = 'keep-alive';
+  options.headers['User-Agent'] = 'Mozilla/5.0 (' + random(options.os) + ')' +
+    ' AppleWebKit/537.36 (KHTML, like Gecko)' +
+    ' Chrome/' + random(options.version) + ' Safari/537.36';
```

Comment:
Basically this still needs to be changed to something like:
```js
' Chrome/' + random(options.version) + (options.mobile ? ' Mobile' : '') + ' Safari/537.36';
```
I'll fix it sometime soon.